src/main/scala/com/fulcrumgenomics/basecalling/ExtractIlluminaRunInfo.scala

```diff
@@ -38,10 +38,11 @@ import dagr.sopt.{arg, clp}
     |The output file will contain a header line and a single line containing the following columns:
     |1. Run Barcode: the unique identifier for the sequencing run and flowcell, stored as "<instrument-name>_<flowcell-barcode>".
     |2. Flowcell Barcode: the flowcell barcode.
-    |3. Run Date: the date of the sequencing run.
-    |4. Read Structure: the description of the logical structure of cycles within the sequencing run, including which cycles
+    |3. Instrument Name: the instrument name.
```

Comment:
I'm just realizing that it would be nice in the usage to have the actual column names (e.g. `instrument_name`) vs. how you have it now.